fortanix-vme/fortanix-vme-runner/src/server.rs

```diff
@@ -0,0 +1,360 @@
+use fnv::FnvHashMap;
```

Comment:
Is this file an accidentally committed old version of what is now in `lib.rs`?

Reply:
Yes it must have been introduced during a rebase. I updated the commit that introduced it